packages/core/src/config/config.ts

```diff
@@ -375,13 +375,17 @@ export class Config {
     const newGeminiClient = new GeminiClient(this);
     await newGeminiClient.initialize(newContentGeneratorConfig);
 
+    const fromGenaiToVertex =
```

Comment:
add a comment explaining why this is needed as this will be surprising to future readers of this code.